File: csharp-netstandard/src/Cloud.Governance.Client/Model/DueDateType.cs


using System;
using System.Collections;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Linq;
using System.IO;
using System.Runtime.Serialization;
using System.Text;
using System.Text.RegularExpressions;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using Newtonsoft.Json.Linq;
using System.ComponentModel.DataAnnotations;
using OpenAPIDateConverter = Cloud.Governance.Client.Client.OpenAPIDateConverter;

namespace Cloud.Governance.Client.Model
{
    
    public enum DueDateType
    {
        None = 0,

        Expired = 1,

        DueToday = 2,

        DueThisWeek = 3,

        DueThisMonth = 4

    }

}

File: typescript/src/models/DueDateType.ts
/* tslint:disable */
/* eslint-disable */
/*
 * Cloud Governance Api
 *
 * Contact: support@avepoint.com
 */

/**
 * 
 * @export
 * @enum {string}
 */
export enum DueDateType {
    None = 0,
    Expired = 1,
    DueToday = 2,
    DueThisWeek = 3,
    DueThisMonth = 4
}

export function DueDateTypeFromJSON(json: any): DueDateType {
    return DueDateTypeFromJSONTyped(json, false);
}

export function DueDateTypeFromJSONTyped(json: any, ignoreDiscriminator: boolean): DueDateType {
    return json as DueDateType;
}

export function DueDateTypeToJSON(value?: DueDateType | null): any {
    return value as any;
}

